key-schedule.go

```diff
@@ -122,21 +122,35 @@ func (nfbks *noFSBaseKeySource) Get(sender leafIndex) []byte {
 	return nfbks.CipherSuite.deriveAppSecret(nfbks.RootSecret, "hs-secret", toNodeIndex(sender), 0, secretSize)
 }
 
+type Bytes1 []byte
+
```

Comment:
nit; may be we can add a one line comment saying : []bytes who header is of size 1